lib/bcdice/game_system/Shiranui.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module BCDice
+  module GameSystem
+    class Shiranui < Base
+      # ゲームシステムの識別子
+      ID = 'Shiranui'
+
+      # ゲームシステム名
+      NAME = '不知火'
+
+      # ゲームシステム名の読みがな
+      SORT_KEY = 'しらぬい'
+
+      # ダイスボットの使い方
+      HELP_MESSAGE = <<~HELP
+        ■∞D66ダイスロール
+        「 ∞D66 」または「 ID66 」
+        （ ID は Infinite D の略です）
+
+        □行動力や攻撃力の指定
+        「 x+∞D66 」または「 x+ID66 」
+        （ x は行動力や攻撃力）
+
+        □鬼火の使用について
+        鬼火を使用する∞D66は、ダイスボットでサポートしていません。
+      HELP
+
+      INFINITE_D66_ROLL_REG = /^((\d+)\+)?(∞|I)D66$/i.freeze
+      register_prefix('(\d+\+)?(∞|I)D66')
+
+      def initialize(command)
+        super(command)
+
+        @sort_barabara_dice = true
+      end
+
+      def eval_game_system_specific_command(command)
+        if (m = INFINITE_D66_ROLL_REG.match(command))
+          fixed_score = m[1].nil? ? nil : m[1].to_i
```

Comment:
```suggestion
          fixed_score = m[1]&.to_i
```

[safe navigation operator `&.`](https://docs.ruby-lang.org/ja/latest/doc/news=2f2_3_0.html) を使うと良さそうです。

Reply:
ありがとうございます。そのまま採用しました。 https://github.com/bcdice/BCDice/pull/533/commits/011051db0b132cf8e70e4c1c182048d9fe267fbf